python/housinginsights/sources/wmata_distcalc.py

```diff
@@ -0,0 +1,295 @@
+        self.mapbox_api_key = {'access_token':self.api_keys['mapbox']['public-token']}
+
+    # def get_data(self):
+    #     self.get_rail_stations
+    #     self.get_bus_stations
+    #     self.get_station_walking_distances
+
+    def get_data(self,output='STANDARD',db='local_database'): 
+        self.get_info()
+        self.get_dist(db=db)
+
+    def get_info(self,output='STANDARD'):
+        self.infoOutput = []
+        #create wmata info header
+        self.infoHeader = ('stop_id_or_station_code','type','name','latitude','longitude','lines')
+        if ( output == 'STANDARD' ):
+            self.infoOutputFileName = self.outputDir + "/wmatainfo.csv"
+            print("Will write WMATA_INFO table to {}".format(self.infoOutputFileName))
+        elif ( output == 'NONE' ):
+            pass
+
+        #Creat objects and write to table
+        self.railStations = self._get_all_rail_stations()
+        self.busStations = self._get_all_bus_stations()
+
+        if ( output == 'STANDARD'):
+            self.result_to_csv(self.infoHeader, self.infoOutput, self.infoOutputFileName)
+        elif ( output == 'NONE'):
+            print("==========================================================================\n")
+            print(self.infoHeader,'\n')
+            print("==========================================================================\n")
+            print(self.infoOutput,'\n')
+
+    def get_dist(self,output='STANDARD',db='local_database'):
+        self.distOutput = []
+        #create wmata info header
+        self.distHeader = ('nlihc_id','type','stop_id_or_station_code','dist_in_miles')
+        if ( output == 'STANDARD' ):
+            self.distOutputFileName = self.outputDir + "/dist.csv"
+            print("Will write WMATA_INFO table to {}".format(self.distOutputFileName))
+        elif ( output == 'NONE' ):
+            pass
+
+        try:
+            engine = dbtools.get_database_engine(db)
+            conn = dbtools.get_database_connection(db) 
+            print("Connected to Housing Insights database")
+            columnset = conn.execute('select column_name from INFORMATION_SCHEMA.COLUMNS where TABLE_NAME=\'project\'')
+            rows = conn.execute('select * from project')
+            conn.close()
+            engine.dispose()
+        except Exception as e:
+            print(e)
+            print("I am unable to connect to the database")
+
+        columns = []
+        for c in columnset:
+            columns.append(c)
+
+        numrow = 0
+        total_rows = rows.rowcount
+
+        wmata_headers = self._get_wmata_headers()
+        railResponse = requests.get("https://api.wmata.com/Rail.svc/json/jStations", headers=wmata_headers)
+        self.railStations = railResponse.json()['Stations']
+
+        for row in rows: #reader:
+            radius = self._get_meters(0.5)
+            self._set_project_info(row,columns)
+
+            numrow = numrow+1
```

Comment:
The cleaner Python way of doing this is to use enumerate on the for loop:

`for idx, row in enumerate(rows):`

Then you don't have to initialize numrow or remember to add 1 to it (easy to forget).